.codespellignore

```diff
@@ -1,10 +1,13 @@
 als
+assertin
 cleary
 dependees
 dur
 extracter
 hist
 inactivate
+indexin
+inpt
```

Comment:
Is there a reason to fix some typos and add others to the ignore? If this is supposed to mean “input” it would be nice to fix.

Reply:
`inPt` may be "input point" - but we could change it in the code to `inPoint` or similar and remove crom the ignore list.